tutorials/tutorial1.ipynb

```diff
@@ -164,862 +175,2019 @@
-    "\n",
-    "Next we read and save the validation data for the model.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-09 20:10:56,268:AtacWorks-bw2h5] Reading intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Read 1260 intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Writing data in 2 batches.\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Extracting data for each batch and writing to h5 file\n",
-      "INFO:2020-06-09 20:10:56,411:AtacWorks-bw2h5] batch 0 of 2\n",
-      "INFO:2020-06-09 20:11:39,943:AtacWorks-bw2h5] Done! Saved to ./Mono.50.2400.val.h5\n"
-     ]
-    }
-   ],
-   "source": [
-    "!python $atacworks/scripts/bw2h5.py \\\n",
-    "           --noisybw dsc.1.Mono.50.cutsites.smoothed.200.bw \\\n",
-    "           --cleanbw dsc.Mono.2400.cutsites.smoothed.200.bw \\\n",
-    "           --cleanpeakbw dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak.bw \\\n",
-    "           --intervals val_intervals.bed \\\n",
-    "           --out_dir ./ \\\n",
-    "           --prefix Mono.50.2400.val \\\n",
-    "           --pad 5000"
+    "```\n",
+    "!atacworks train\n",
+    "        --cleanbw <path to folder containing all clean bigwig files for training> \\\n",
+    "        --noisybw <path to folder containing all clean bigwig files for validation> \\\n",
+    "        --cleanpeakfile <path to folder containing all clean peak files>\n",
+    "        --out_home <out-dir> \\\n",
+    "        --distributed \\\n",
+    "        --config <path-to-custom-config-if-any> \\\n",
+    "        --config_mparams <path-to-custom-model-structure-if-any>\n",
+    "```"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This produces a .h5 file in the current directory (`Mono.50.2400.val.h5`) containing the validation data for the model.\n",
-    "\n",
-    "\n",
-    "## Step 7: Train and validate a model using the parameters in the given config files\n",
-    "\n",
-    "We next train an AtacWorks model to learn a mapping from the noisy (50-cell) ATAC-seq signal to the clean (2400-cell) ATAC-seq signal and peak calls. The two .yaml files that we downloaded into the `configs` directory contain all the parameters that describe how to train the model. `configs/model_structure.yaml` contains parameters that control the architecture of the model and  `configs/config_params.yaml` contains parameters that control the process of training, such as the learning rate and batch size.\n",
-    "\n",
-    "To train the model, we supply the training and validation datasets as well as the two config files."
+    "To change any of the parameters for training, copy paste the relevant default config files at `$atacworks/configs` to the current location."
```

Comment:
Let's move this section on config files up and title it as a new Appendix 1. Then the current appendix 1 -> appendix 2 and appendix 2 -> appendix 3.